### ISSUE TITLE
more precision on "not production ready"

### ISSUE THREAD

**Issue by allan-simon** [tatsuo-ishii/docker-pgpool-II-pgpool_setup#1]
Hello, thanks to provide this repository I'm a bit curious about the sentence 

```
Caution: This container is not suitable for production purpose.
```

is there some stuff that one should improve in order to make it production, or it's more a warning that you're providing 0 warranty it will actually work ?

**Comment by tatsuo-ishii** [comment]
Hi,
The reason why the container is not ready for production is, it is based on a script "pgpool_setup", which deployes  pgpool-II and PostgreSQL servers inside the container. This is not good because 1) the performance is terrible because PostgreSQL has to use slow file system inside the container and 2) you cannot enjoy the benefit of pgpool-II's high-availability and performance boost by using load balancing since those will need distributing PostgreSQL servers to different hosts.
pgpool_setup is deigned to be used for evaluation or testing purposes (actually it is used for the regression test suit coming with the pgpool-II source distribution).

In summary the container should work well if used for testing or evaluation purposes but not suitable for production use.

In the mean time I have a plan to create a pgpool-II container which can be used for production. stay tuned.

**Comment by allan-simon** [comment]
> In the mean time I have a plan to create a pgpool-II container which can be used for production. stay tuned.

awesome, today I've been doing some research on how to create a cluster of dockerized postgresql , managed with kubernetes and that's how I found repo, but it seems that there's currently no "prior art" on doing so.
something mixing your to-be-created container with 
https://zwischenzugs.wordpress.com/2015/04/06/play-with-kubernetes-quickly-using-docker/
and 
http://patg.net/mysql,kubernetes,coreos,docker,vmware/2015/03/27/mysql-replication-kubernetes/
